raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -1149,12 +1187,58 @@ def handle_lock_expired(
     return TransitionResult(mediator_state, events)
 
 
+def handle_node_change_network_state(
+        mediator_state: MediatorTransferState,
+        state_change: ActionChangeNodeNetworkState,
+        channelidentifiers_to_channels: typing.ChannelMap,
+        pseudo_random_generator: random.Random,
+        block_number: typing.BlockNumber,
+) -> TransitionResult:
+    """ If a certain node comes online:
+    1. Check if a channel exists with that node
+    2. Check that this channel is a route, check if the route is valid.
+    3. Check that the transfer was stuck because there was no route available.
+    4. Send the transfer again to this now-available route.
+    """
+    if state_change.network_state != NODE_NETWORK_REACHABLE:
+        return TransitionResult(mediator_state, list())
+
+    # Get route whose node_address matches the newly reachable node
+    try:
+        route = next(route for route in mediator_state.routes
```

Comment:
I believe this is breaking our [style guidelines](https://github.com/raiden-network/raiden/blob/919b167e1aee1b0a6803297f7090baf05de8ee07/CONTRIBUTING.md#general-style). If not, then we should discuss which style to use and make it consistent:

```python
func(a
     b)  # lisp style, with arguments aligned at the next column after the opening parens

func(  # multiline arguments, each on it's own line, indented by 4 additional spaces
    a,
    b,
)
```

Reply:
yes it should have a line break after the parentheses